src/borg/helpers/parseformat.py

```diff
@@ -536,6 +536,17 @@ def validator(text):
     return validator
 
 
+def relative_time_marker_validator(text: str):
+    day_offset_regex = r"^\d+d$"
+    month_offset_regex = r"^\d+m$"
+    day_match = re.compile(day_offset_regex).search(text)
+    month_match = re.compile(month_offset_regex).search(text)
+    if not day_match and not month_match:
```

Comment:
This could be simplified by using `r"^\d+[md]$"`.

That even has the advantage that we could also add y h m s easily.